ui/app/components/clients/horizontal-bar-chart.js

```diff
@@ -183,28 +173,15 @@ export default class HorizontalBarChart extends Component {
         const hoveredElement = actionBars.filter((bar) => bar[labelKey] === data[labelKey]).node();
         this.tooltipTarget = hoveredElement;
         this.isLabel = false;
-        this.tooltipText = this.total
```

Comment:
removed percent because tooltip is getting pretty full with added sync counts
<img width="177" alt="Screenshot 2024-01-17 at 12 13 55 PM" src="https://github.com/hashicorp/vault/assets/68122737/84895db8-9c40-41a1-be0e-3a25883624bb">